common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -188,3 +188,9 @@ def definition_to_xml(self, resource_fs):
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
         return xml_object
+
+    @property
+    def non_editable_metadata_fields(self):
+        non_editable_fields = super(SequenceDescriptor, self).non_editable_metadata_fields
+        non_editable_fields.extend([SequenceFields.is_entrance_exam, ])
```

Comment:
The dividing line for me between settings and content has to do with when you would expect it to be edited. In particular, `Scope.settings` varies over usage (which with Courses really means individual runs of the course). So, if you'd expect `is_entrance_exam` to change between course runs (like dates do), then it's probably a `Scope.settings`. Otherwise, it's `Scope.content`.

Another question to ask (less applicable to courses), is whether, if you had two copies of the same object in the course, should the value be shared or different? (For instance, you might have the same problem as a practice problem and in an exam, but with different grading in each, so you would expect the grading to be a setting).